manifests/weave/base/faucet.yaml

```diff
@@ -24,7 +24,7 @@ spec:
             secretKeyRef:
               name: weave_placeholder-faucet-settings
               key: FAUCET_MNEMONIC
-        - name: FAUCET_CREDIT_AMOUNT_TKN
+        - name: FAUCET_CREDIT_AMOUNT_IOV
```

Comment:
Q: this does not look backwards compatible with older networks. Can you do this in kissnet overlay instead?

Reply:
Actually it was a bug, so this needs to be done where it's done. 
Turns out we were using defaults all this time :)

Reply:
@alpe ^^ so I think this is good to go.

Reply:
@webmaster128 let me know if I'm right about this? I assume this env variable should have been `_IOV` and not `_TKN` since the first devnet here?

Reply:
The last component of the env variable name must match the token ticker. `FAUCET_CREDIT_AMOUNT_TKN` affects token `TKN` only. Since we never had such a token, the env variable did nothing before.

Setting `FAUCET_CREDIT_AMOUNT_IOV` is correct for all networks where the faucet provides IOV tokens.

Reply:
@alpe FYI ^^. Let's merge as is then?